### ISSUE TITLE
Regarding PyTorch TPU

### ISSUE THREAD

**Issue by vainaixr** [HighCWu/SelfGAN#2]
Hello,

I think TPU is not actually being used, we need to use xla for using google colab TPU.

This example shows how to do it.
https://github.com/pytorch/xla/blob/master/contrib/colab/PyTorch_TPU_XRT_1_13.ipynb
They have given example for mnist, is it possible to figure out how to train GAN with it? 

Thanks

**Comment by HighCWu** [comment]
Yes, the version of the pytorch model I provided is only the cuda version, and the model version of keras uses the tpu version.
I have been paying attention to the xla project of pytorch for a long time. At that time, some operations were not supported. Maybe it can be used now, but I don't have time for it, you can try it yourself.
The good news is that colab has upgraded their GPUs, and TPUs are sometimes very troublesome to use.
